src/app/constants/game.ts

```diff
@@ -5,6 +5,8 @@ export const DEFAULT_STORAGE_CAPACITY = 600
 
 export const START_POPULATION = 100
 
-export const EASY_ATTACK_COUNTDOWN_IN_SECONDS = 10 * 60
-export const MEDIUM_ATTACK_COUNTDOWN_IN_SECONDS = 0.5 * 60 // TODO: change to 8 after testing
-export const HARD_ATTACK_COUNTDOWN_IN_SECONDS = 5 * 60
\ No newline at end of file
+export const EASY_ATTACK_COUNTDOWN_IN_MILISECONDS = 10 * 60 * 1000
+export const MEDIUM_ATTACK_COUNTDOWN_IN_MILISECONDS = 0.5 * 60 * 1000 // TODO: change to 8 after testing
```

Comment:
What should be changed to 8?

Reply:
I mean that here only for testing is set to 30s but later we have to change it, I had here 8 minutes but we can then select another times for these constants.